python/tvm/relay/frontend/tensorflow.py

```diff
@@ -2895,16 +2909,25 @@ def _parse_import_prerequisites(self, graph):
                 which are not supported
         """
         missing_operators = set()
+        from tensorflow.python.framework import op_def_registry
         for node in graph.node:
+            getOpDef = op_def_registry._registered_ops.get if hasattr(op_def_registry,\
+                        "_registered_ops") else op_def_registry.get
+            op_def = getOpDef(node.op)
             if node.op == "Placeholder" or node.op == 'PlaceholderWithDefault':
                 pass
             elif node.op == "Const":
                 pass
+            elif node.op in ["PartitionedCall", "StatefulPartitionedCall"]:
+                pass
             else:
                 if any([node.op in t for t in [_identity_list, _convert_map,
                                                _convert_map_rnn,
                                                _control_flow_nodes]]):
                     pass
+                elif op_def is not None and op_def.is_stateful:
+                    self._main_graph_proto._stateful_ops_list.append(node.op)
```

Comment:
No need of another list. If needed you may append (StatufulOperator) to node.op

Reply:
Removed the new list. only adding these stateful ops to missing operator list.